web-components/src/components/tooltip/Tooltip.test.ts

```diff
@@ -34,52 +34,52 @@ describe("Tooltip", () => {
     expect(true).toBeTruthy();
   });
 
-  test("should show/hide Tooltip Component", async () => {
-    const element: Tooltip = await fixtureFactory("Test text", false);
-    element.showTooltip();
-    await elementUpdated(element);
+  // test("should show/hide Tooltip Component", async () => {
```

Comment:
So now do we need these tests to remain or can we clear them out and move coverage to the Theme component?